e2e-pw/src/oss/poms/saved-views.ts

```diff
@@ -0,0 +1,376 @@
+import { expect, Locator, Page } from "src/oss/fixtures";
+import {
+  Color,
+  ColorList,
+  updatedView,
+  updatedView2,
+} from "../specs/smoke-tests/saved-views.spec";
```

Comment:
!! I'd argue that this is an anti-pattern. Imports between POMs and specs should be one-way, as in, only specs get to import from POMs.

I think all of these imports can be passed as function args, no?